src/js/components/Layout/Breadcrumbs.jsx

```diff
@@ -1,11 +1,70 @@
-import React from 'react';
+/* eslint-disable react/no-array-index-key */
+import React, { Component } from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { showModal } from '../../actions';
 
-const Breadcrumbs = () => (
-  <div className="bradcrumbs-container d-flex">
-    <span>
-      Breadcrumbs
-    </span>
-  </div>
-);
+class Breadcrumbs extends Component {
+  constructor(props) {
+    super(props);
 
-export default Breadcrumbs;
+    this.openModal = this.openModal.bind(this);
+  }
+
+  componentDidMount() {
+
+  }
+
+  openModal() {
+    this.props.showModal();
+  }
+
+  render() {
+    const listItems = window.location.pathname.split('/');
```

Comment:
I have seen that in the grails part, it was done using the current controller and current action. As our URL are in this format controller/action, I think it will work well. 
But it's a proposition, this approach hasn't been approved yet @jmiranda